share/sass/gwdetchar-omega.scss

```diff
@@ -27,6 +27,43 @@ body {
 		margin-bottom: 120px;
 		min-height: 100%;
 		font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+		font-weight: 300;
+}
+
+h1 {
+		font-weight: normal;
+}
+
+h2 {
```

Comment:
@alurban, you probably want to simplify this:

```sass
h1, h2, h3, h4, h5, h6 {
		font-weight: normal;
}
```
or define a custom class (e.g. backporting from bootstrap v4):

```sass
.font-weight-normal {
		font-weight: normal;
}
```

Reply:
Aha! I didn't know you could do that, I'll try again shortly.

Reply:
@duncanmmacleod, thanks for the suggestion, I have made this change and verified that works with the Hanford example (linked to above).